extensions-contrib/ambari-metrics-emitter/src/test/java/org/apache/druid/emitter/ambari/metrics/AmbariMetricsEmitterConfigTest.java

```diff
@@ -81,8 +81,8 @@ public void testSerDeDruidToTimelineEventConverter() throws IOException
         new DefaultObjectMapper()
     );
     serde = mapper.readerFor(DruidToTimelineMetricConverter.class)
-                                          .readValue(mapper.writeValueAsBytes(
-                                              whiteListBasedDruidToTimelineEventConverter));
+                  .readValue(mapper.writeValueAsBytes(
```

Comment:
This is still not formatted properly. If the argument(s) break, the closing paren should also break. But for this code it would be preferred to extract `mapper.writeValueAsBytes(whiteListBasedDruidToTimelineEventConverter)` as a variable.

But not critical for tests for bothering with another commit, I'll merge.